lib/modules/settings/models/notification_settings.dart

```diff
@@ -86,6 +123,52 @@ class IqamaSettings {
       };
 }
 
+class PrayerNotificationSettings {
+  PrayerNotificationSettings({
+    required this.athanSettings,
+    required this.iqamaSettings,
+    required this.preAthanSettings,
+  });
+
+  PrayerNotificationSettings.defaultValue()
+      : this(
+          athanSettings: AthanSettings(
+              enabled: true,
+              notificationSettings: NotificationSettings(
+                sound: true,
+                vibration: false,
+                ringtone: athanRingtones[0],
+              )),
+          iqamaSettings: IqamaSettings(
+            enabled: true,
+            delay: 10,
+            notificationSettings: NotificationSettings(
+              sound: true,
+              vibration: false,
+              ringtone: iqamaRingtones[0],
+            ),
+          ),
```

Comment:
it would be nice to implement `default` for these as well, it seems a bit off one only has default

Reply:
Are you planning to fix this in this PR? otherwise we can create an issue with general code fixes, beside this the PR is good

Reply:
Not in this pr but hopefully soon